yt/go/yt/integration/cypress_client_test.go

```diff
@@ -261,6 +263,28 @@ func (s *Suite) TestLinkNode(t *testing.T, yc yt.Client) {
 	require.Equal(t, yt.NodeLink, typ)
 }
 
+func (s *Suite) TestConcatNode(t *testing.T, yc yt.Client) {
+	t.Parallel()
+	var err error
+
+	ctx, cancel := context.WithTimeout(s.Ctx, time.Second*30)
+	defer cancel()
+
+	src1 := tmpPath()
+	src2 := tmpPath()
+	dst := tmpPath()
+
+	_, err = yc.CreateNode(ctx, dst, yt.NodeTable, nil)
+	require.NoError(t, err)
+	_, err = yc.CreateNode(ctx, src1, yt.NodeTable, nil)
+	require.NoError(t, err)
+	_, err = yc.CreateNode(ctx, src2, yt.NodeTable, nil)
+	require.NoError(t, err)
+
+	err = yc.ConcatNode(ctx, []ypath.YPath{src1, src2}, dst, nil)
+	require.NoError(t, err)
```

Comment:
Maybe the test should have a few rows in the source tables and also check that the contents match?

Reply:
I tried, but it does not seem possible atm: integration test calls both rpc-client and http-client. rpc-client does not implement WriteTable :|

Reply:
okay, seems like it's possible to skip RPC tests, added one

Reply:
you could use `yttest.Env.YT` from `Suite` - this is generally available http client
with it you can always do `write_table` via http, and not skip tests for RPC implementation